### PR TITLE
Add labels from dataplane

### PR DESCRIPTION
**Pull request by fao89**
It will make it possible to filter all pods from specific DataplaneDeployment
ref: https://github.com/openstack-k8s-operators/dataplane-operator/pull/545

### CODE REVIEW

**Review by fultonj — COMMENTED** [review]
+2

File: controllers/openstack_ansibleee_controller.go [hunk 1]
```diff
@@ -428,12 +422,16 @@ func (r *OpenStackAnsibleEEReconciler) jobForOpenStackAnsibleEE(ctx context.Cont
...
 // labelsForOpenStackAnsibleEE returns the labels for selecting the resources
 // belonging to the given openstackansibleee CR name.
-func labelsForOpenStackAnsibleEE(name string, deployIdentifier string) map[string]string {
-	return map[string]string{
+func labelsForOpenStackAnsibleEE(name string, labels map[string]string) map[string]string {
+	ls := map[string]string{
 		"app":                   "openstackansibleee",
-		"deployIdentifier":      deployIdentifier,
 		"openstackansibleee_cr": name,
+		"aee":                   "true",
```
Comment (slagle) [inline controllers/openstack_ansibleee_controller.go:429]: What is the aee label for? The shortname is actually osaee, is it meant to correspond to that?

Comment (fao89) [reply controllers/openstack_ansibleee_controller.go:429]: I just wanted to have a shorter label than `app=openstackansibleee`,
but you are right, better use the shortname

**Review by slagle — APPROVED** [review]

**Comment by fao89** [comment]
recheck

**Pull request merged**